twill11/hometask-4/my_hash/my_hash.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# Implementation of my own hash
+class MyHash
+  include Enumerable
+
+  ONE = 1
+
+  def initialize
+    @hash_variable = []
+  end
```

Comment:
Read about what is Hash.
Your hash must contain a hash function that calculates the index from the key.